src/components/FlyoutMenu/FlyoutMenu.test.jsx

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { fireEvent, render, waitFor, waitForElement } from '@testing-library/react';
+
+import FlyoutMenu, { FlyoutMenuDirection } from './FlyoutMenu';
+
+const applyButtonText = 'Apply';
+const cancelButtonText = 'Cancel';
+
+describe('FlyoutMenu', () => {
+  it('Renders an open transactional menu', async () => {
+    const { queryByText } = render(
+      <FlyoutMenu defaultOpen transactional triggerId="flyout-test" />
```

Comment:
We gonna get console warnings about missing required `iconDescription` for these tests